### PR TITLE
Helper fixes

### PR DESCRIPTION
**Pull request by proullon**

### CODE REVIEW

File: helpers/helpers.go [hunk 1]
```diff
@@ -218,7 +288,7 @@ func goType(pkg string, f *descriptor.FieldDescriptorProto) string {
 		}
 		return "byte"
 	case descriptor.FieldDescriptorProto_TYPE_ENUM:
-		return fmt.Sprintf("*%s.%s", pkg, shortType(*f.TypeName))
```
Comment (jhayotte) [inline helpers/helpers.go]: Why did you remove the point here?

Comment (proullon) [inline helpers/helpers.go]: I thought the ```if pkg != "" {
 			pkg = pkg + "."
 		}``` was in the top of the function
